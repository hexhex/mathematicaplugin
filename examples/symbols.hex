matrix(1,1,"a"). matrix(1,2,"b").
matrix(2,1,"c"). matrix(2,2,"d").

b(1,"e"). b(2,"f").

vektor(X) :- &linearSolve[matrix,b](X).
