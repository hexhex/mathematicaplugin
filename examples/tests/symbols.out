{vektor(1,1,"Times Power Plus Times -1 b c Times a d -1 Plus Times d e Times -1 b f"), vektor(2,1,"Times Power Plus Times b c Times -1 a d -1 Plus Times c e Times -1 a f"), b(1,"e"), b(2,"f"), matrix(1,1,"a"), matrix(1,2,"b"), matrix(2,1,"c"), matrix(2,2,"d")}
